### PR TITLE
TELCODOCS-365: Entry for CNI pod-level bonding and TP table

### PR DESCRIPTION
**Pull request by tmulquee**
Jira ticket is https://issues.redhat.com/browse/TELCODOCS-365

For 4.10.

https://deploy-preview-39478--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-pod-level-bonding

https://deploy-preview-39478--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-technology-preview

### CODE REVIEW

**Comment by tmulquee** [comment]
@SchSeba @mmirecki can you review the RN content in this pull request.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -63,6 +63,17 @@ This release adds improvements related to the following components and concepts.
...
 
+[id="ocp-4-10-pod-level-bonding"]
+==== Pod-level bonding for secondary networks
+Bonding at the pod level is vital to enable workloads inside pods that require high availability. With pod-level bonding, you can create a bond interface from multiple virtual function interfaces (single root I/O virtualization (SR-IOV) in kernel mode interface). The virtual functions are passed into the pod and attached to a kernel driver.
+
+Scenarios where pod level bonding is required include creating a bond interface from multiple virtual functions on different physical functions. Creating a bond interface from two different interfaces on the host can be used to achieve high availability at pod level.
+
+[NOTE]
+====
+Link Aggregation Control Protocol (LACP) mode is not supported, as is the case for {product-title}.
```
Comment (SchSeba) [inline release_notes/ocp-4-10-release-notes.adoc:74]: I didn't get this point `as is the case for {product-title}.`

**Comment by tmulquee** [comment]
That's a variable that's interpreted by the docs build as "Red Hat
OpenShift Platform".
Best
/tonym

**Comment by tmulquee** [comment]
Sorry, Red Hat OpenShift Container Platform.

**Comment by tmulquee** [comment]
@mmirecki @SchSeba @cgoncalves 
Hi folks, 
I need SME and QE reviews so we can get the RN lgtms needed to merge this text.

**Comment by tmulquee** [comment]
Sebastian Scheinkman12:41 PM
sriov (single...) virtual function
Sebastian Scheinkman12:42 PM
interfaces - > physical funtions
You12:43 PM
Creating a bond interface from two different physical functions on the host can be used to achieve high availability at pod level.

**Comment by tmulquee** [comment]
@SchSeba @mmirecki  Edits have been put through. Can you please check if I've missed anything? If not, please add an /lgtm
each.

**Comment by evgenLevin** [comment]
lgtm

**Comment by tmulquee** [comment]
Requested merge.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1055,7 +1065,11 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
+<<<<<<< HEAD
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:1068]: @tmulquee this is not ready for merge. you have committed merge bars

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -284,6 +284,16 @@ You can use the enhancement to run packet captures on multiple nodes at the same
 A new `--node-selector` argument provides a way to identify which nodes you are collecting packet captures for.
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
+[id="ocp-4-10-pod-level-bonding"]
+==== Pod-level bonding for secondary networks (Technology Preview)
+Bonding at the pod level is vital to enable workloads inside pods that require high availability and more throughput. With pod-level bonding, you can create a bond interface from multiple single root I/O virtualization (SR-IOV) virtual function interfaces in kernel mode interface). The SR-IOV virtual functions are passed into the pod and attached to a kernel driver.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:289]: in the kernel mode interface?

remove ) after interface

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -284,6 +284,16 @@ You can use the enhancement to run packet captures on multiple nodes at the same
...
+
+Scenarios where pod level bonding is required include creating a bond interface from multiple SR-IOV virtual functions on different physical functions. Creating a bond interface from two different physical functions on the host can be used to achieve high availability at pod level.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:291]: pod-level bonding

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -284,6 +284,16 @@ You can use the enhancement to run packet captures on multiple nodes at the same
...
+
+[NOTE]
+====
+The current functionality of Bond CNI is available only in active-backup mode - for further details, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2037214[*BZ#2037214*].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:295]: mode. For further details,

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-10-release-notes.adoc:289]: @tmulquee These issues are still unresolved. Thanks!

**Pull request merged**